packages/crypto-lib/src/base/index.ts

```
@@ -1,51 +1,48 @@
+    buffer[j] = tmp;
+    j--;
+  }
+  return buffer;
 }
 
 export function concatBytes(b1: Uint8Array | Buffer, b2: Uint8Array | Buffer): Uint8Array {
-    return utils.concatBytes(Uint8Array.from(b1), Uint8Array.from(b2))
+  return utils.concatBytes(Uint8Array.from(b1), Uint8Array.from(b2));
 }
 
 export function randomBytes(length: Number): Buffer {
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)